pkg/services/cloudmigration/cloudmigrationimpl/snapshot_mgmt.go

```diff
@@ -159,6 +167,14 @@ func (s *Service) getMigrationDataJSON(ctx context.Context, signedInUser *user.S
 		})
 	}
 
+	// Notification Policy can only be managed by updating its entire tree, so we send the whole thing as one item.
+	migrationDataSlice = append(migrationDataSlice, cloudmigration.MigrateDataRequestItem{
+		Type:  cloudmigration.NotificationPolicyType,
+		RefID: notificationPolicies.Name, // no UID available
```

Comment:
What is the "ID" we use internally to store the policies in that case? Is it just `Name`?

Reply:
@Proximyst Yes; when creating the resource in the target instance, we won't use it, since the API itself doesn't take any uid/names, and the GMS backend might not use the ID at all in some other cases (resources that have names in their API path), but I preferred to pass something for bookkeeping.